test_config.py

```diff
@@ -44,6 +44,8 @@
 mkl_link_opt_lin = ["-lmkl_intel_ilp64", "-lmkl_sequential", "-lmkl_core",
                     "-lOpenCL", "-lmkl_sycl", "-lpthread", "-ldl"]
 
+mkl_cuda_link_lin = ["-lonemkl", "-lonemkl_blas_cublas", "-lpthread", "-ldl", "-lOpenCL"]
```

Comment:
According to https://github.com/oneapi-src/oneMKL/blob/develop/README.md , it seems only one library is needed to be specified in cmd line.
```suggestion
mkl_cuda_link_lin = ["-lonemkl"]
```